src/main/java/com/redhat/red/build/finder/JSONUtils.java

```diff
@@ -40,7 +45,7 @@ public static String dumpString(Object obj) {
             String s = mapper.writerWithDefaultPrettyPrinter().writeValueAsString(obj);
             return s;
         } catch (JsonProcessingException e) {
-            e.printStackTrace();
```

Comment:
I wonder if we should propagate these exceptions up?

Reply:
I don't know how much sense this makes, but because it's a utility method, I preferred to return null rather than throw an exception because it's easier to handle, but I guess that might not be very Java-like.

Reply:
The general idea of the change was to print these to the log and not the active terminal.

Reply:
I think that due to the size of the log being so large its going to be easy to miss problems. Further, (although not specific to this method) I think that if there is an error that  might affect future processing it should be cascaded up. However perhaps this should be discussed for a separate issue outside of this PR?